presets/4.4/tune/supafly_fpv/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt

```diff
@@ -4,105 +4,283 @@
-set gyro_lpf2_static_hz = 1000
-set gyro_lpf1_dyn_min_hz = 300
-set gyro_lpf1_dyn_max_hz = 600
-set dyn_notch_count = 3
-set dyn_notch_q = 300
-set dyn_notch_min_hz = 100
+set simplified_gyro_filter_multiplier = 120
+set gyro_lpf1_static_hz = 0
+set gyro_lpf1_dyn_min_hz = 0
+set gyro_lpf1_dyn_max_hz = 0
+set dyn_notch_count = 5
+set dyn_notch_q = 400
+set dyn_notch_min_hz = 90
 set dyn_notch_max_hz = 800
 
-# -- Dterm filtering --
-
-set simplified_dterm_filter = OFF
+set simplified_dterm_filter_multiplier = 140
 set dterm_lpf1_dyn_min_hz = 0
 set dterm_lpf1_dyn_max_hz = 0
 set dterm_lpf1_dyn_expo = 0
-set dterm_lpf1_static_hz = 80
-set dterm_lpf2_static_hz = 140
-
-#$ OPTION BEGIN (CHECKED): RPM filtering
-    set motor_pwm_protocol = DSHOT600
-    set dshot_bidir = ON
-    set simplified_gyro_filter = OFF
-    set rpm_filter_harmonics = 1
-    set gyro_lpf2_static_hz = 1000
-    set gyro_lpf1_dyn_min_hz = 300
-    set gyro_lpf1_dyn_max_hz = 600
-    set gyro_lpf1_dyn_expo = 8
-    set dyn_notch_count = 2
-    set dyn_notch_q = 400
-    set dyn_notch_min_hz = 100
-    set dyn_notch_max_hz = 800
-#$ OPTION END
-
-#$ OPTION BEGIN (CHECKED): 4s Lipo with HD Cam (2400-2700kv motors)
-    # -- Sliders --
-    set simplified_master_multiplier = 120
-    set simplified_i_gain = 75
-    set simplified_d_gain = 110
-    set simplified_pi_gain = 110
-    set simplified_dmax_gain = 0
-    set simplified_feedforward_gain = 90
-    set simplified_pitch_d_gain = 95
-    set simplified_pitch_pi_gain = 115
-    simplified_tuning apply
-#$ OPTION END
-
-#$ OPTION BEGIN (UNCHECKED): 4s Lipo with No HD Cam (2400-2700kv motors)
-    # -- Sliders --
-    set simplified_master_multiplier = 120
-    set simplified_i_gain = 75
-    set simplified_d_gain = 110
-    set simplified_pi_gain = 110
-    set simplified_dmax_gain = 0
-    set simplified_feedforward_gain = 90
-    set simplified_pitch_d_gain = 90
-    simplified_tuning apply
-#$ OPTION END
-
-#$ OPTION BEGIN (UNCHECKED): 6s Lipo with HD Cam (1750 - 1950kv motors)
-    # -- Sliders --
-    set simplified_master_multiplier = 100
-    set simplified_i_gain = 75
-    set simplified_d_gain = 110
-    set simplified_pi_gain = 115
-    set simplified_dmax_gain = 0
-    set simplified_feedforward_gain = 90
-    set simplified_pitch_pi_gain = 110
-    simplified_tuning apply
-#$ OPTION END
-
-#$ OPTION BEGIN (UNCHECKED): 6s Lipo with No HD Cam (1750 - 1950kv motors)
-    # -- Sliders --
-    set simplified_master_multiplier = 100
-    set simplified_i_gain = 75
-    set simplified_d_gain = 110
-    set simplified_pi_gain = 115
-    set simplified_dmax_gain = 0
-    set simplified_feedforward_gain = 90
-    set simplified_pitch_d_gain = 95
-    simplified_tuning apply
-#$ OPTION END
+
+#$ OPTION_GROUP BEGIN: Filters (default to MPU6000/BMI/ICM206xx Gyro)
+
+    #$ OPTION BEGIN (CHECKED): RPM filtering (Uncheck for Dynamic Notch Filter Setup)
+    #$ INCLUDE_WARNING: misc/warnings/en/rpm_filters.txt
```

Comment:
same for warning here. Plz move it up to the header before description
Or can make your own warning text with "#$ WARNING: ...."

Reply:
ok